web/server.js

```diff
@@ -158,9 +159,28 @@ app.get('/', (req, res) => {
     }
     const { locale } = res;
     const fullUrl = 'https://www.electricitymap.org' + req.originalUrl;
+
+    // basic auth and time machine cookie if we're in the back in time container
```

Comment:
Can we change the comment to something even a contributor could understand
Something along the lines of:
// Basic auth for premium access